common/migration/1635880160929-AddUpdatedAtInCOC.ts

```diff
@@ -0,0 +1,148 @@
+        await queryRunner.query(`ALTER TABLE "screener" ALTER COLUMN "authTokenSent" DROP DEFAULT`);
+        await queryRunner.query(`COMMENT ON COLUMN "screener"."authTokenSent" IS NULL`);
+        await queryRunner.query(`ALTER TABLE "screener" ALTER COLUMN "authToken" DROP DEFAULT`);
+        await queryRunner.query(`COMMENT ON COLUMN "screener"."authToken" IS NULL`);
+        await queryRunner.query(`ALTER TABLE "screener" ALTER COLUMN "verifiedAt" DROP DEFAULT`);
+        await queryRunner.query(`COMMENT ON COLUMN "screener"."verifiedAt" IS NULL`);
+        await queryRunner.query(`ALTER TABLE "screener" ALTER COLUMN "verification" DROP DEFAULT`);
+        await queryRunner.query(`COMMENT ON COLUMN "screener"."verification" IS NULL`);
+        await queryRunner.query(`ALTER TABLE "match" ALTER COLUMN "dissolveReason" DROP DEFAULT`);
+        await queryRunner.query(`COMMENT ON COLUMN "match"."dissolveReason" IS NULL`);
+        await queryRunner.query(`ALTER TABLE "pupil" ALTER COLUMN "teacherEmailAddress" DROP DEFAULT`);
+        await queryRunner.query(`COMMENT ON COLUMN "pupil"."teacherEmailAddress" IS NULL`);
+        await queryRunner.query(`ALTER TABLE "pupil" ALTER COLUMN "lastUpdatedSettingsViaBlocker" DROP DEFAULT`);
+        await queryRunner.query(`COMMENT ON COLUMN "pupil"."lastUpdatedSettingsViaBlocker" IS NULL`);
+        await queryRunner.query(`ALTER TABLE "pupil" ALTER COLUMN "authTokenSent" DROP DEFAULT`);
+        await queryRunner.query(`COMMENT ON COLUMN "pupil"."authTokenSent" IS NULL`);
+        await queryRunner.query(`ALTER TABLE "pupil" ALTER COLUMN "authToken" DROP DEFAULT`);
+        await queryRunner.query(`COMMENT ON COLUMN "pupil"."authToken" IS NULL`);
+        await queryRunner.query(`ALTER TABLE "pupil" ALTER COLUMN "verifiedAt" DROP DEFAULT`);
+        await queryRunner.query(`COMMENT ON COLUMN "pupil"."verifiedAt" IS NULL`);
+        await queryRunner.query(`ALTER TABLE "pupil" ALTER COLUMN "verification" DROP DEFAULT`);
+        await queryRunner.query(`COMMENT ON COLUMN "pupil"."verification" IS NULL`);
+        await queryRunner.query(`ALTER TABLE "pupil_tutoring_interest_confirmation_request" ALTER COLUMN "reminderSentDate" DROP DEFAULT`);
+        await queryRunner.query(`COMMENT ON COLUMN "pupil_tutoring_interest_confirmation_request"."reminderSentDate" IS NULL`);
+        await queryRunner.query(`ALTER TABLE "project_match" ALTER COLUMN "dissolveReason" DROP DEFAULT`);
+        await queryRunner.query(`COMMENT ON COLUMN "project_match"."dissolveReason" IS NULL`);
+        await queryRunner.query(`ALTER TABLE "certificate_of_conduct" RENAME COLUMN "updatedAt" TO "inspectingScreenerId"`);
+        await queryRunner.query(`ALTER TABLE "certificate_of_conduct" ADD CONSTRAINT "FK_05729540b3678f1917321b621bf" FOREIGN KEY ("inspectingScreenerId") REFERENCES "screener"("id") ON DELETE NO ACTION ON UPDATE NO ACTION`);
```

Comment:
Have you upgraded TypeORM locally? These migrations look strange ...

Reply:
The version both locally and remote is ~0.29 so i don't think the version is different. I don't remember changing anything else in the schema either. I agree, These do look strange when compared to other migrations. I'll dig deeper and see where all these ALTER statements are coming from. Maybe Postgres versions being different might cause this? I don't know.